diff --git a/src/Makefile.in b/src/Makefile.in
index 9ffb29d..83e6659 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -1449,7 +1449,7 @@ code-from-errno.h: mkerrcodes Makefile
 
 errnos-sym.h: Makefile mkstrtable.awk errnos.in
 	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=2 -v nogettext=1 \
-		-v prefix=GPG_ERR_ -v namespace=errnos_ \
+		-v prefix=GPG_ERR_ -v libgpg_error_namespace=errnos_ \
 		$(srcdir)/errnos.in >$@
 
 mkheader: mkheader.c Makefile
