diff -Naur a/third_party/cpuinfo/src/src/riscv/linux/riscv-hw.c b/third_party/cpuinfo/src/src/riscv/linux/riscv-hw.c
--- a/third_party/cpuinfo/src/src/riscv/linux/riscv-hw.c	2024-03-16 12:12:29.835536873 +0000
+++ b/third_party/cpuinfo/src/src/riscv/linux/riscv-hw.c	2024-03-16 12:06:55.888004794 +0000
@@ -14,7 +14,6 @@
 #endif
 
 #include <sched.h>
-#include <sys/hwprobe.h>
 
 #include <cpuinfo/log.h>
 #include <riscv/api.h>
