Index: chromium-129.0.6668.58/third_party/cpuinfo/BUILD.gn
===================================================================
--- chromium-129.0.6668.58.orig/third_party/cpuinfo/BUILD.gn
+++ chromium-129.0.6668.58/third_party/cpuinfo/BUILD.gn
@@ -105,6 +105,12 @@ source_set("cpu_specific") {
     ]
   }
 
+  if (current_cpu == "riscv64") {
+    sources = [
+      "src/src/riscv/uarch.c",
+    ]
+  }
+
   # Because "init.c" is reused again, but only has sources in the corresponding
   # build config.
   deps = [ ":x86_cache" ]
@@ -191,6 +197,14 @@ source_set("cpu_and_os_specific") {
     }
   }
 
+  if (is_linux && current_cpu == "riscv64") {
+    sources = [
+      "src/src/riscv/linux/init.c",
+      "src/src/riscv/linux/riscv-hw.c",
+      "src/src/riscv/linux/riscv-isa.c",
+    ]
+  }
+
   if (is_android && (current_cpu == "x86" || current_cpu == "x64")) {
     sources = [
       "src/src/x86/linux/cpuinfo.c",

