Index: chromium-135.0.7049.52/third_party/swiftshader/src/Reactor/BUILD.gn
===================================================================
--- chromium-135.0.7049.52.orig/third_party/swiftshader/src/Reactor/BUILD.gn
+++ chromium-135.0.7049.52/third_party/swiftshader/src/Reactor/BUILD.gn
@@ -307,7 +307,7 @@ if (supports_subzero) {
 
 if (supports_llvm) {
   swiftshader_source_set("swiftshader_llvm_reactor") {
-    llvm_dir = "../../third_party/llvm-10.0"
+    llvm_dir = "../../third_party/llvm-16.0"
 
     deps = [
       ":swiftshader_reactor_base",
Index: chromium-135.0.7049.52/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn
===================================================================
--- chromium-135.0.7049.52.orig/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn
+++ chromium-135.0.7049.52/third_party/swiftshader/third_party/llvm-16.0/BUILD.gn
@@ -778,6 +778,7 @@ swiftshader_llvm_source_set("swiftshader
     "llvm/lib/MC/MCContext.cpp",
     "llvm/lib/MC/MCDXContainerStreamer.cpp",
     "llvm/lib/MC/MCDXContainerWriter.cpp",
+    "llvm/lib/MC/MCDisassembler/MCDisassembler.cpp",
     "llvm/lib/MC/MCDisassembler/MCRelocationInfo.cpp",
     "llvm/lib/MC/MCDwarf.cpp",
     "llvm/lib/MC/MCELFObjectTargetWriter.cpp",

